mlrun/runtimes/nuclio/function.py

```diff
@@ -510,8 +509,9 @@ def add_v3io_stream_trigger(
                 **kwargs,
             ),
         )
-        self.spec.min_replicas = shards
-        self.spec.max_replicas = shards
+        if shards:
+            self.spec.min_replicas = shards
+            self.spec.max_replicas = shards
```

Comment:
What if shards is None? Does min/max replicas get a default value later in the flow? (or in Nuclio?🫣)